governance/roles.md

```diff
@@ -87,21 +89,8 @@ Leads, any Chair may act as Technical Lead.
   * Asking for new proposals to be made to address an identified need.
   * Partnering with Technical Leads to establish a roadmap and manage ongoing
     projects.
-  * Serving as Security TAG leadership representative to ensure the project and
-    project lead(s) is successful
-    * check in with the project lead regularly to discuss progress, blockers,
-      and updates
-    * provide mentorship to project lead(s)
-    * ensure the schedule set by the project lead(s) is adhered to
-    * verify the scope and proposed deliverables of the project are in alignment
-      with the [Charter](charter.md) prior to recommendation of becoming a
-      project
-    * provide the TOC and others on the leadership team with updates
-    * keep the issue up to date for the project lead(s) if they don't have write
-      access to do so
-    * enforce and encourage company diversity on a given project where possible
-    * provide guardrails and guidance to the project and project lead(s) as
-      appropriate
+ * Chairs may additionally perform any actions of technical leads as needed,
+   especially serving as the Security TAG leadership representative to a project.
```

Comment:
```suggestion
   especially serving as the Security TAG leadership representative to a project and approving Pull Requests especially for top-level content of the repository. 
```

Reply:
I don't disagree but this point probably warrants  a line item/bullet point, rather than being part of additional actions a chair may perform.